Trunk.toml

```diff
@@ -11,6 +11,12 @@ dist = "dist"
 public_url = "/"
 # Whether to include hash values in the output file names.
 filehash = true
+# Full text of script block including starting and ending markers
+pattern_script = """
+<script type="module">
+console.log("This will be injected into the HTML artifact")
+</script>
+"""
```

Comment:
How does this fit into the system? Would this actually be used by the Trunk CLI? This PR does not appear to be handling it.

Reply:
So the top of the file explains that this is supposed to be a sample `Trunk.toml` that demos all the options, but there are some that are undocumented, including `pattern_script` . I needed to use this to get my project working and thought it would be helpful to add to the example `Trunk.toml` file. I'm happy to remove it if it's undesirable or misleading :)
https://github.com/thedodd/trunk/blob/16ba0353822bed5e84d174cadbe02d4659e5a630/src/config/models.rs#L296